src/client/App.js

```diff
@@ -40,6 +41,13 @@ function App() {
           <Route exact path="/channel/:id" isAuthenticated={isAuthenticated}>
             <Channel />
           </Route>
+          <Route
+            exact
+            path="/channel/:id/about"
```

Comment:
I think usually we would use `plural` for the routes, such as  `channels` instead of `channel`. Nothing wrong with it just usually a good convention 👍 
```suggestion
            path="/channels/:id/about"
```